src/utility/Language/SystemLanguage.h

```diff
@@ -35,24 +35,24 @@ namespace Gamebuino_Meta {
 const MultiLang lang_keyboard_type[] = {
 	{LangCode::en, "type"},
 	{LangCode::de, "tippen"},
-	{LangCode::fr, "�crire"},
+	{LangCode::fr, "écrire"},
 	{LangCode::es, "tipo"},
 	{LangCode::nl, "typ"},
 };
 
 const MultiLang lang_keyboard_back[] = {
 	{LangCode::en, "back"},
-	{LangCode::de, "zur�ck"},
+	{LangCode::de, "zurück"},
 	{LangCode::fr, "retour"},
-	{LangCode::es, "espald"},
+	{LangCode::es, "regresar"},
```

Comment:
this can't extend six characters, else it won't fit into the GUI thing correctly :/